src/App.tsx

```diff
@@ -3,29 +3,99 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getToods } from './api/api';
 
 interface State {
   selectedUserId: number;
+  todos: Todo[];
+  checkedTodo: number,
 }
 
 class App extends React.Component<{}, State> {
   state: State = {
     selectedUserId: 0,
+    todos: [],
+    checkedTodo: 0,
+  };
+
+  componentDidMount() {
+    this.loader();
+  }
+
+  loader = async () => {
+    await getToods
```

Comment:
use await or then , not together
read the difference between them one more time, I think you don't understand it now
https://learn.javascript.ru/promise-basics#potrebiteli-then-catch-finally
https://learn.javascript.ru/async-await

Reply:
thanks a lot for the articles